### ISSUE TITLE
Multi class images

### ISSUE THREAD

**Issue by BirATMAN** [Asad-Ismail/Grounding-Dino-FineTuning#9]
hi, do i not need bounding box annotation and object-specific label/class name for training datasets with multi-object images? how should the text input be?

**Comment by Asad-Ismail** [comment]
Hello thank you for your interest in the work you do need box annotaiton and class names for training dataset, I have added the sample dataset that you can test with https://drive.google.com/file/d/1D2qphEE98Dloo3fUURRnsxaIRw076ZXX/view?usp=drive_link
